src/azure-cli/azure/cli/command_modules/backup/_validators.py

```diff
@@ -10,10 +10,10 @@
 
 def datetime_type(string):
     """ Validates UTC datettime in accepted format. Examples: 31-12-2017 """
-    accepted_date_formats = ['%d-%m-%Y']
+    accepted_date_formats = ['%d-%m-%Y', '%d-%m-%Y-%H:%M:%S']
```

Comment:
keep the 2 error messages separate. Given a scenario, the user might just want to see one format as the correct format

Reply:
Need to discuss